rtv/content.py

```diff
@@ -8,19 +8,30 @@
 
 from .errors import SubmissionURLError, SubredditNameError
 
+is_utf8 = True
+
 
 def clean(unicode_string):
     """
-    Convert unicode string into ascii-safe characters.
+    Checks if -a was given, if it was it
+    converts unicode string into ascii-safe characters.
     """
+    if not is_utf8:
+        if six.PY2:
+            ascii_string = unicode_string.encode('ascii', 'replace')
+        else:
+            ascii_string = unicode_string.encode().decode('ascii', 'replace')
 
-    if six.PY2:
-        ascii_string = unicode_string.encode('ascii', 'replace')
+            ascii_string = ascii_string.replace('\\', '')
```

Comment:
this line was already here on line 22 on the original, I didn't know why it was there, so I kept it